src/content/doc-sdk-golang/methods/raw-query.mdx

```diff
@@ -1,18 +1,18 @@
 ---
 sidebar_position: 1
-sidebar_label: RawQuery
-title: Golang | SDK | Methods | RawQuery
+sidebar_label: QueryRaw
+title: Golang | SDK | Methods | QueryRaw
 description: This method runs a set of SurrealQL statements against the database, based on a raw SurrealQL query. 
 ---
 
 import Label from "@components/shared/Label.astro";
 
-# `.RawQuery()` {#query}
+# `.QueryRaw()` {#query}
 
 Runs a set of SurrealQL statements against the database, based on a raw SurrealQL query.
 
 ```go title="Method Syntax"
-db.RawQuery(sql string, vars map[string]interface{})
+db.QueryRaw(sql string, vars map[string]interface{})
```

Comment:
[QueryRaw](https://pkg.go.dev/github.com/surrealdb/surrealdb.go#QueryRaw) is a function and not a method on the db type. Also the parameters are different than specified here:

```suggestion
func QueryRaw(db *DB, queries *[]QueryStmt) error
```